node/packages/parser/src/ethereum/yearn.ts

```diff
@@ -21,6 +21,11 @@ export class Parser implements GenericParser {
     if (!data) return
 
     const txSigHash = getSigHash(data)
+    const isSupportedSigHash = txSigHash
```

Comment:
Does this provide us with anything that an undefined `abiInterface` doesn't? I also noticed we have the `getAbiInterface` switch wrapped in an unneeded anon function while looking.

Reply:
You're right, there is actually no value added there - removed in https://github.com/shapeshift/unchained/pull/288/commits/2e555c474ee86bd86212c34b7f271124a9fc3a86